zdm/survey.py

```diff
@@ -556,14 +806,10 @@ def geometric_lognormals(lmu1,ls1,lmu2,ls2,bins=None,
     
 def make_widths(s:Survey,state):
     """
+    WHAT DOES THIS METHOD DO?
```

Comment:
This method takes a distribution of intrinsic FRB widths (lognormal, defined by wlogmean and wlogsigma), and returns a list of w_i, p(w_i), where the w_i are i=1...N values of width, and p(w_i) are statistical weights associated with each. The \sum_i p(w_i) should sum to unity always. Each w_i is used to calculate a separate efficiency table.

Reply:
updated